libs/arcade-cli/arcade_cli/main.py

```diff
@@ -271,16 +271,11 @@ def mcp(
             handle_cli_error("Failed to run MCP server")
 
     except KeyboardInterrupt:
-        console.print("\n[yellow]MCP server stopped[/yellow]")
-        raise typer.Exit(0)
+        console.print("\n[yellow]MCP server gracefully shutdown[/yellow]")
```

Comment:
Since `arcade mcp` is a long running process and keyboard interrupt is the only way to gracefully shutdown, we don't want to raise a typer exit. This ensures that the event is correctly sent to posthog as a 'success'